installlib/installcommand.class.php

```diff
@@ -55,7 +55,11 @@ protected function configure() {
 			->setName('install')
 			->setDescription('FreePBX Installation Utility')
 			;
-
+		// sleazy, probably should be in in a constructor
+		if (PHP_OS == 'FreeBSD') {
+			$this->settings['webroot']['default'] =
```

Comment:
you can define this during install. The fact that you redefine it and hardcode a path later on will confuse anyone using install cli parameters